src/Console/Command/ScopeHintCommand.php

```diff
@@ -20,8 +20,8 @@ class ScopeHintCommand extends Command
 
     public function __construct(
         ScopeHintService $scopeHintService,
-        string $name = null)
-    {
+        string $name = null
+    ) {
```

Comment:
@avstudnitz So the curley brackets follow the brackets if the parameters have multiply lines. and if not like with the function configure() in line 32 the curley brackets are in a new line?

Reply:
@a-leurs Yes, exactly.